lib/orogen/spec/task_context.rb

```diff
@@ -638,6 +632,15 @@ def configuration_object(klass, name, type, default_value)
                 klass.new(self, name, type, default_value)
             end
 
+            # @api private
+            #
+            # Resolve a type name into a type object, and leave type object alone
+            def resolve_type_if_needed(type)
+                return type if type.kind_of?(Class) && type <= Typelib::Type
```

Comment:
what is `<=` operation between Class "type instances"?

Reply:
It tests that `type` is a subclass of Typelib::Type or Typelib::Type itself (reminds the `class Subclass < ParentClass` syntax)